src/usermod.c

```diff
@@ -1062,7 +1063,8 @@ process_flags(int argc, char **argv)
 				}
 				dflg = true;
 				user_newhome = optarg;
-				if (user_newhome[0] != '/') {
+				/* If no empty string, check whether the path is absolute */
```

Comment:
I think I would remove that comment.  It doesn't say anything that the code doesn't say.

If anything needs to be clarified further, I'd do it in the commit message.

Reply:
This is of course true, but written text can be understood more quickly than code if it is a little more complicated. In the end, of course, it's a matter of opinion and taste. Personally, I hate source code without comments.

Reply:
> This is of course true, but written text can be understood more quickly than code if it is a little more complicated.

Then the goal is to never write code that is more complicated.  That's a very difficult thing, actually, but doable, IMO.  :-)